<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	font: 22px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font:30px;
	color:#444444;
}</string>
    </property>
    <property name="text">
     <string>实验三：最短连线</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>261</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>设置栏</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>1.方格大小：（a×b）</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>2.点数设置：</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>可斜线</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">line_option</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>仅直线</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">line_option</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>84</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>4.连线规则:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>3.自定义坐标：（x,y）</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="data_num">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>y:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="data_y">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>44</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>180</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>44</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(注：默认坐标随机生成)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>加入</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="data_x">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>44</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>180</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="data_b">
     <property name="geometry">
      <rect>
       <x>133</x>
       <y>70</y>
       <width>44</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_1">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>113</x>
       <y>70</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>b:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="data_a">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>70</y>
       <width>44</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>a:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>451</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>方格图</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>29</y>
       <width>411</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="fig_con"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>751</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>控制台</string>
    </property>
    <widget class="QTextBrowser" name="res_console">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>711</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生成最短连线</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="line_option"/>
 </buttongroups>
</ui>
